routes/notification.router.js

```diff
@@ -0,0 +1,73 @@
+const express = require('express');
+const notificationService = require('../services/notification.service');
+
+const router = express.Router();
+
+router.get('/:id', async (req, res) => {
+  const { id } = req.params;
+  try {
+    const foundNotification = await notificationService.getNotification(id);
+    if (!foundNotification) {
+      res.status(400).json({ message: `Notification ${id} doesn't exist` });
+    } else {
+      res.status(200).send(foundNotification);
+    }
+  } catch (err) {
+    console.error(err);
+    res.status(400).json({ error: err });
+  }
+});
+
+router.get('/', async (req, res) => {
```

Comment:
We actually don't really need this route (or any of the routes in this file). The only two routes we need is 1) for a user to get all of their own notifications, and 2) for a user to close a notification (so the notification gets deleted). The notifications will be created in other parts of the backend such as when an admin requests changes on someone's log, but we don't need a route for that.